packages/visual-editor/src/runtime/sideboard-runtime.ts

```diff
@@ -312,8 +310,10 @@ class SideboardRuntimeImpl
                 })
               );
             }
+          }
+          if (result.state === "valid") {
+            secrets[key] = result.grant.access_token;
           } else {
-            result.state satisfies "signedout";
```

Comment:
was there a reason to remove this type assertion? the purpose of it is to ensure that if we add a new kind of state, there would be a compile-error until we handle it somehow here.

Reply:
javascript switches don't enforce exhaustiveness, so the latest change doesn't actually have any effect. a satisfies type assertion would still be required.

Reply:
yeah, but isn't that the line you removed?

Reply:
oh sorry, my bad, I missed part of the diff, the assertion is in fact there.